host/xtest/asym_cipher_perf.c

```diff
@@ -0,0 +1,938 @@
+	fprintf(stderr, "                   RSAES_PKCS1_V1_5_ENCRYPT, RSAES_PKCS1_V1_5_DECRYPT\n");
+	fprintf(stderr, "                   RSAES_PKCS1_OAEP_SHA1_ENCRYPT, RSAES_PKCS1_OAEP_SHA1_DECRYPT\n");
+	fprintf(stderr, "                   RSAES_PKCS1_OAEP_SHA224_ENCRYPT, RSAES_PKCS1_OAEP_SHA224_DECRYPT\n");
+	fprintf(stderr, "                   RSAES_PKCS1_OAEP_SHA256_ENCRYPT, RSAES_PKCS1_OAEP_SHA256_DECRYPT\n");
+	fprintf(stderr, "                   RSAES_PKCS1_OAEP_SHA384_ENCRYPT, RSAES_PKCS1_OAEP_SHA384_DECRYPT\n");
+	fprintf(stderr, "                   RSAES_PKCS1_OAEP_SHA512_ENCRYPT, RSAES_PKCS1_OAEP_SHA512_DECRYPT\n");
+	fprintf(stderr, "                   RSASSA_PKCS1_V1_5_SHA1_SIGN, RSASSA_PKCS1_V1_5_SHA1_VERIFY\n");
+	fprintf(stderr, "                   RSASSA_PKCS1_V1_5_SHA224_SIGN, RSASSA_PKCS1_V1_5_SHA224_VERIFY\n");
+	fprintf(stderr, "                   RSASSA_PKCS1_V1_5_SHA256_SIGN, RSASSA_PKCS1_V1_5_SHA256_VERIFY\n");
+	fprintf(stderr, "                   RSASSA_PKCS1_V1_5_SHA384_SIGN, RSASSA_PKCS1_V1_5_SHA384_VERIFY\n");
+	fprintf(stderr, "                   RSASSA_PKCS1_V1_5_SHA512_SIGN, RSASSA_PKCS1_V1_5_SHA512_VERIFY\n");
+	fprintf(stderr, "                   RSASSA_PKCS1_PSS_MGF1_SHA1_SIGN, RSASSA_PKCS1_PSS_MGF1_SHA1_VERIFY\n");
+	fprintf(stderr, "                   RSASSA_PKCS1_PSS_MGF1_SHA224_SIGN, RSASSA_PKCS1_PSS_MGF1_SHA224_VERIFY\n");
+	fprintf(stderr, "                   RSASSA_PKCS1_PSS_MGF1_SHA256_SIGN, RSASSA_PKCS1_PSS_MGF1_SHA256_VERIFY\n");
+	fprintf(stderr, "                   RSASSA_PKCS1_PSS_MGF1_SHA384_SIGN, RSASSA_PKCS1_PSS_MGF1_SHA384_VERIFY\n");
+	fprintf(stderr, "                   RSASSA_PKCS1_PSS_MGF1_SHA512_SIGN, RSASSA_PKCS1_PSS_MGF1_SHA512_VERIFY\n");
+	fprintf(stderr, "                   ECDSA_SIGN, ECDSA_VERIFY, ECDH, X25519\n");
+	fprintf(stderr, "  -l LOOP          Inner loop iterations [%u]\n", l);
+	fprintf(stderr, "  -n LOOP          Outer test loop iterations [%u]\n", n);
+	fprintf(stderr, "  -r|--random      Get input data from /dev/urandom (default: all zeros)\n");
+	fprintf(stderr, "  -k SIZE          Plaintext Length [%u]\n", size);
+	fprintf(stderr, "  -d WIDTH_BITS    ECC:the width_bits only support 192/224/256/384/521 [%u]\n", width_bits);
+	fprintf(stderr, "                   DH: width_bits <= 2048, RSA: width_bits <= 4096 [%u]\n", width_bits);
+	fprintf(stderr, "  -s SALT_LEN      Salt length in bytes(only when ALGO is one of RSA SSA_PKCS1_PSS_*)[%u]\n", salt_len);
+	fprintf(stderr, "  -w|--warmup SEC  Warm-up time in seconds: execute a busy loop before [%d]\n", warmup);
+	fprintf(stderr, "  -v               Be verbose (use twice for greater effect)\n");
+}
+
+static void allocate_shm(TEEC_SharedMemory *shm, size_t sz)
+{
+	TEEC_Result res = TEEC_ERROR_GENERIC;
+
+	shm->buffer = NULL;
+	shm->size = sz;
+	res = TEEC_AllocateSharedMemory(&ctx, shm);
+	check_res(res, "TEEC_AllocateSharedMemory", NULL);
```

Comment:
s/TEEC_AllocateSharedMemory/TEEC_AllocateSharedMemory()/ to hightlight it's a function label.

Ditto at lines 251, 271, 293, 319 and 535.